### ISSUE TITLE
warnings, errors, tracebacks while generating docs during RPM build process

### ISSUE THREAD

**Issue by TomasTomecek** [fedora-modularity/conu#90]
```
+ sleep 1
+ sphinx-build docs/source html
Running Sphinx v1.6.5
making output directory...
WARNING: The config value `version' has type `module', defaults to `str'.
WARNING: The config value `release' has type `module', defaults to `str'.
loading pickled environment... not yet created
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 15 source files that are out of date
updating environment: 15 added, 0 changed, 0 removed
reading sources... [100%] reference/util_index
/home/tt/g/conu/conu-0.1.0/docs/source/reference/docker_container.rst:6: WARNING: autodoc: failed to import class u'DockerContainer' from module u'conu'; the following exception was raised:
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/sphinx/ext/autodoc.py", line 665, in import_object
    obj = self.get_attr(obj, part)
  File "/usr/lib/python2.7/site-packages/sphinx/ext/autodoc.py", line 554, in get_attr
    return safe_getattr(obj, name, *defargs)
  File "/usr/lib/python2.7/site-packages/sphinx/util/inspect.py", line 185, in safe_getattr
    raise AttributeError(name)
AttributeError: DockerContainer
/home/tt/g/conu/conu-0.1.0/docs/source/reference/docker_container.rst:9: WARNING: autodoc: failed to import class u'DockerRunCommand' from module u'conu'; the following exception was raised:
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/sphinx/ext/autodoc.py", line 665, in import_object
    obj = self.get_attr(obj, part)
  File "/usr/lib/python2.7/site-packages/sphinx/ext/autodoc.py", line 554, in get_attr
    return safe_getattr(obj, name, *defargs)
  File "/usr/lib/python2.7/site-packages/sphinx/util/inspect.py", line 185, in safe_getattr
    raise AttributeError(name)
AttributeError: DockerRunCommand
/home/tt/g/conu/conu-0.1.0/docs/source/reference/docker_container.rst:12: WARNING: autodoc: failed to import class u'DockerContainerFS' from module u'conu'; the following exception was raised:
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/sphinx/ext/autodoc.py", line 665, in import_object
    obj = self.get_attr(obj, part)
  File "/usr/lib/python2.7/site-packages/sphinx/ext/autodoc.py", line 554, in get_attr
    return safe_getattr(obj, name, *defargs)
  File "/usr/lib/python2.7/site-packages/sphinx/util/inspect.py", line 185, in safe_getattr
    raise AttributeError(name)
AttributeError: DockerContainerFS
/home/tt/g/conu/conu-0.1.0/docs/source/reference/docker_image.rst:6: WARNING: autodoc: failed to import class u'DockerImage' from module u'conu'; the following exception was raised:
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/sphinx/ext/autodoc.py", line 665, in import_object
    obj = self.get_attr(obj, part)
  File "/usr/lib/python2.7/site-packages/sphinx/ext/autodoc.py", line 554, in get_attr
    return safe_getattr(obj, name, *defargs)
  File "/usr/lib/python2.7/site-packages/sphinx/util/inspect.py", line 185, in safe_getattr
    raise AttributeError(name)
AttributeError: DockerImage
/home/tt/g/conu/conu-0.1.0/docs/source/reference/docker_image.rst:9: WARNING: autodoc: failed to import class u'DockerImageFS' from module u'conu'; the following exception was raised:
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/sphinx/ext/autodoc.py", line 665, in import_object
    obj = self.get_attr(obj, part)
  File "/usr/lib/python2.7/site-packages/sphinx/ext/autodoc.py", line 554, in get_attr
    return safe_getattr(obj, name, *defargs)
  File "/usr/lib/python2.7/site-packages/sphinx/util/inspect.py", line 185, in safe_getattr
    raise AttributeError(name)
AttributeError: DockerImageFS
/home/tt/g/conu/conu-0.1.0/docs/source/reference/probe.rst:4: WARNING: autodoc: failed to import class u'Probe' from module u'conu'; the following exception was raised:
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/sphinx/ext/autodoc.py", line 665, in import_object
    obj = self.get_attr(obj, part)
  File "/usr/lib/python2.7/site-packages/sphinx/ext/autodoc.py", line 554, in get_attr
    return safe_getattr(obj, name, *defargs)
  File "/usr/lib/python2.7/site-packages/sphinx/util/inspect.py", line 185, in safe_getattr
    raise AttributeError(name)
AttributeError: Probe
/home/tt/g/conu/conu-0.1.0/docs/source/reference/util_filesystem.rst:4: WARNING: autodoc: failed to import class u'Directory' from module u'conu'; the following exception was raised:
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/sphinx/ext/autodoc.py", line 665, in import_object
    obj = self.get_attr(obj, part)
  File "/usr/lib/python2.7/site-packages/sphinx/ext/autodoc.py", line 554, in get_attr
    return safe_getattr(obj, name, *defargs)
  File "/usr/lib/python2.7/site-packages/sphinx/util/inspect.py", line 185, in safe_getattr
    raise AttributeError(name)
AttributeError: Directory
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] reference/util_index
generating indices... genindex py-modindex
writing additional pages... search
copying static files... WARNING: html_static_path entry u'/home/tt/g/conu/conu-0.1.0/docs/source/_static' does not exist
done
copying extra files... done
dumping search index in English (code: en) ... done
dumping object inventory... done
build succeeded, 10 warnings.
+ rm -rf html/.doctrees html/.buildinfo
+ exit 0
```

**Comment by dhodovsk** [comment]
Generating docs requires conu to be installed (or in PYTHONPATH) so it can import names for classes reference api docs.

**Comment by TomasTomecek** [comment]
Fixed by #99.
